handlers/email/email.go

```diff
@@ -146,6 +165,9 @@ func defaultFormatFunc(email *Email) Formatter {
 
 // Log handles the log entry
 func (email *Email) Log(e log.Entry) {
+	if !email.send {
```

Comment:
There is a race here, this needs to be surrounded by `email.rw.Rlock()` or changed to us atomic, which in this case might make more sense.